.github/workflows/website.yml

```diff
@@ -46,12 +46,19 @@ jobs:
         with:
           name: 'Data'
 
+      - name: Run the test that generates the plots report.
+        run: |
+          pytest tests/IVIMmodels/unit_tests/test_ivim_fit.py --json-report
```

Comment:
I think `json-report` is now a requirement?

Also, why run this test again and not get the same data from the initial run?

Reply:
> I think json-report is now a requirement?

yeah, done.

> Also, why run this test again and not get the same data from the initial run?

Do you mean the unit test run? I don't find a straight way of doing it, as we have a matrix of runs so which file we should get the data from those multiple jobs?
<img width="442" alt="image" src="https://github.com/user-attachments/assets/d82e7553-b425-498b-9efb-8ed9d67aaaf3">

Reply:
What about saving from the first run only? https://stackoverflow.com/questions/70532207/github-action-yaml-how-to-refer-an-item-in-a-matrix-array

But also only if it's one that runs the analysis. Maybe that's too much optimization.